### DISCUSSION TITLE
Random System Sounds - shell 1.8 beta

### DISCUSSION THREAD

**Discussion by hiranokite** [moudey/Shell#51] [General]
OS: Windows 11 22H2 22621.755

I'm getting some random windows background sounds while using shell 1.8 beta, it comes from windows process %windir%\System32\taskhostw.exe and play the sound %windir%\Media\dm\Windows Foreground.wav, i cant figure out what do to play the sound but only occurs while shell 1.8 beta x64 is active.

Tested on shell 1.7 x64 and the random sound don't play.
Is this a know issue?

![random-sound](https://user-images.githubusercontent.com/5361459/200214197-aa524f30-0897-4624-a946-6e1fdc588565.png)

**Comment by moudey** [comment]
This is a notice to close the File Explorer handle. It will be deprecated in the next update.
